css/index.css

```diff
@@ -1,3 +1,489 @@
+    padding: 5px;
+    border-radius: 5%;
+    width: 150px;
+
+
+}
+
+button {
+    border: 1px solid black;
+}
+
+.features-about {
+    display: flex;
+    justify-content: space-around;
+    margin: 30px 0px;
+    line-height: 20px;
+}
+
+.services-vision {
+    display: flex;
+    justify-content: space-around;
+    margin: 30px 0px;
+    line-height: 20px;
+}
+
+.services {
+    margin: 0px 20px 0px 0px;
+}
+
+.product {
+    margin: 0px 20px 0px 0px;
+}
+
+.features {
+    display: flex;
+    flex-direction: column;
+    margin-right: 20px;
+
+}
+
+.about {
+    display: flex;
+    flex-direction: column;
+
+}
+
+.main-header header {
+    display: flex;
+    align-items: center;
+    justify-content: space-between;
+}
+
+.main-header {
+    padding: 30px 0;
+
+}
+
+.section {
+    display: flex;
+    justify-content: space-between;
+}
+
+.sectiion2 {
+    display: flex;
+}
+
+.footer {
+    text-align: center;
+    margin: 20px;
+}
+
+.container1 {
+    display: flex;
+}
+
+.container2 {
+    display: flex;
+}
+
+.container3 {
+    display: flex;
+}
+
+.container4 {
+    display: flex;
+
+}
+
+.box {
+    border: 1px solid black;
+    margin: 10px;
+    background-color: lightgray;
+    display: block;
+    width: 50%;
+    padding: 10px
+}
+
+.facts {
+    margin: 0px 40px 0px 0px;
+}
+
+ul {
+    margin: 20px 0px;
+}
+
+ul li {
+    margin: 20px 20px;
+    list-style: disc;
+}
+
+.middle-img {
+    width: 100%;
+}
+
+@media(max-width: 800px) {
+    header {
+        flex-direction: column;
+    }
+
+    .header-img {
+        display: none;
+    }
+
+    .logo {
+        order: 1;
+    }
+
+    nav {
+        order: 2;
+    }
+
+    .demand-main {
+        justify-content: center;
+    }
+
+    header {
+        display: flex;
+        justify-content: center;
+    }
+}
+
+@media(max-width: 500px) {
+    header {
+        flex-direction: column;
+    }
+
+    .header-img {
+        display: none;
+    }
+
+    .logo {
+        order: 1;
+    }
+
+    nav {
+        order: 2;
+    }
+
+    .demand-main {
+        justify-content: center;
+    }
+
+    header {
+        display: flex;
+        justify-content: center;
+    }
+
+    nav {
+        flex-direction: column;
+    }
+
+    nav a {
+        padding: 10px;
+        border-bottom: 1px solid lightgray;
+        border-top: 1px solid lightgray;
+        width: 500px;
+        padding-left: 225px;
+
+    }
+
+    .features-about {
+        flex-direction: column;
+    }
+
+    .features {
+        order: 1;
+        margin-bottom: 30px;
+    }
+
+    .about {
+        order: 2;
+        margin-bottom: 30px;
+    }
+
+    .services-vision {
+        flex-direction: column;
+    }
+
+    .services {
+        order: 1;
+        margin-bottom: 30px;
+    }
+
+    .product {
+        order: 2;
+        margin-bottom: 30px;
+    }
+
+    .vision {
+        order: 3;
+    }
+
+    .contact {
+        margin-top: 30px;
+    }
+}
+/* Services page */
+@media(max-width: 800px) {
```

Comment:
Is the breakpoint code repeated here for services? If both HTML files are using the same CSS file, there's no need. If you named and classed common elements from pages the same way, the CSS will affect both.